### ISSUE TITLE
GOAL-4: the ngboost reactivation gate names pure_alpha, and no repo's main computes it — the gate is unfalsifiable as written

### ISSUE THREAD

**Issue by hallovorld** [hallovorld/renquant-orchestrator#729]
Follow-on to #725, which found that GOAL-4's second ensemble head (ngboost) is not
unexplored but **rejected**, with a written reactivation gate:

> flag stays off until either **pure_alpha ≥ +0.04** or an alternative sizing scheme
> proves itself

This round asks the obvious next question: **can that gate be evaluated?**

### `pure_alpha` has no producer [本次实测 2026-08-01]

`git grep -il -E "pure[_ -]?alpha"` against each repo's **`origin/main`** tree:

| repo | hits | where |
|---|---|---|
| renquant-model | **0** | — |
| renquant-pipeline | **0** | — |
| renquant-backtesting | **0** | — |
| renquant-strategy-104 | 6 | only inside `_disable_reason` **string values**, in all 3 configs |
| renquant-orchestrator | 1 | one roadmap doc |

Checked structurally, not by grep alone: in all three strategy configs `pure_alpha`
appears **as a key: nowhere**; it occurs only inside two `_disable_reason` strings.

So no code in model, pipeline or backtesting computes or persists the quantity the gate
names. The reactivation condition is **unfalsifiable as written** — not wrong, but not
checkable either, which is why the head has sat off since 2026-05-09 with nothing able to
move it.

The natural home would have been the gate stamp's `alpha_economics` block; it is not
there. That field on the served prod artifact carries a different thing entirely:

```
alpha_economics = {"passed": true,
                   "reason": "no benchmark sleeve closed trades; full WF metrics are alpha-owned",
                   "evidence": {"n_benchmark_sleeve_closed": 0}}
```

### A narrowing I had to make mid-check

I first saw a second disabled feature — `ranking.panel_scoring.quality_floor.edge_sharpe_floor`
— also mentioning pure-alpha and was about to report *"two gates rest on an unmeasurable
quantity."* Reading it properly:

> 2026-05-09 (E52 audit): pure-alpha ceiling on current panel ~+0.029, no τ achieves
> target FDR=0.30 OR 0.40 … **Re-enable when val μ-IC ≥ +0.04 + placebo |IC| < 0.01.**

Its gate is stated in **val μ-IC and placebo IC**, which this system does compute.
Pure-alpha appears there as *diagnosis*, not as the gate. **Only ngboost's gate names the
unproduced quantity.** Recording the near-miss because "a number in the same sentence as
the gate" is not the gate.

### An open question I am NOT answering

Both reasons are dated 2026-05-09 and quote a pure-alpha figure: ngboost's component
`~+0.018`, and E52's *panel ceiling* `~+0.029`. The ngboost bar is `+0.04`. If those two
figures are commensurable, the bar sits **above the panel's own recorded ceiling**, and
the gate would be not merely unmeasurable but unreachable on that panel.

I cannot establish commensurability — one is described as a component, the other as a
ceiling, and neither has a definition I can read. This needs whoever ran E52/E55, and it
is the difference between "instrument the metric" and "the gate was never satisfiable".

### Asks

1. Define `pure_alpha` somewhere executable, or restate ngboost's gate in a quantity that
   already has a producer (val μ-IC + placebo, as E52's gate does).
2. Answer the commensurability question above before anyone invests in measuring it.

### Not claimed

That the ngboost head should be re-enabled — #725 records it losing a 27-month A/B by
−3.78 APY pts / −0.14 Sharpe, and nothing here revisits that. That `pure_alpha` was never
computed historically; E55 evidently produced a number for it. What is measured is that
**nothing on any current `main` produces it today.**

### Method note

My first search ran against my local sibling checkouts and returned 0 everywhere. Those
checkouts are stale — model **73** commits behind `origin/main`, pipeline **58**,
backtesting **53** — so that result established nothing. Every number above was re-run
with `git grep` against the fetched `origin/main` of each repo.

**Comment by hallovorld** [comment]
**Second disconnected field in the same config node — and it makes the picture coherent.**

This issue found the ngboost reactivation gate (`pure_alpha ≥ +0.04`) names a quantity no
repo produces. Measured today, the *whole node* is wired to nothing `[本次实测 2026-08-01]`.

`configs/strategy_config.json` → `panel_ltr.ngboost`:

```
enabled          : True
artifact_path    : artifacts/prod/ngboost-head.alpha158_fund.json
min_val_mu_ic    : 0
params           : {n_estimators: 400, learning_rate: 0.01}
training_notes   : "Stage-2 NGBoost head for residual returns"
```

Field by field:

| field | state |
|---|---|
| `enabled: True` | nothing trains it — **0** ngboost mentions in the 2026-08-01 retrain log; the artifact is frozen at **2026-05-22 (71 days)** |
| `min_val_mu_ic: 0` | **zero readers.** `git grep` over `origin/main` of orchestrator / model / pipeline / backtesting / strategy-104: matches only the **3 strategy-104 config files** themselves |
| `_disable_reason` (sibling serving node) | names `pure_alpha ≥ +0.04` — no producer, per this issue |

So the ensemble's second head is governed by a training flag that trains nothing, a
threshold of **0** that nothing reads, and a reactivation gate stated in a quantity nothing
computes. Three fields, three dead ends, one node.

### Positive control, because a zero has to be earned

A `0` from a search is worthless unless the search can find things. Same command, same
refs `[本次实测]`:

```
model    min_val_mu_ic          files=0     model    dependence_aware_mean  files=26
pipeline min_val_mu_ic          files=0     model    lag_alignment          files=23
                                            pipeline shadow_health          files=10
                                            pipeline ApplyShadowScoringTask files=9
```

The instrument finds four control strings across both repos and returns 0 for this one, so
the zero is a measurement rather than a failed query.

### One thing I checked and it is NOT the answer

`mu_ic` *does* have a producer — `renquant-model`
`src/renquant_model_common/acceptance_entry_ic.py:69` returns
`(rank_ic, mu_ic, sigma_ic, n_paired)`. But it computes an **entry IC from a trades
frame**, not a validation-set μ-IC over a scoring panel. I am **not** treating it as the
missing producer for `min_val_mu_ic`; establishing whether the two are the same estimand
needs whoever wrote E52, and assuming they are would be the substitution error this
programme keeps paying for.

### Not claimed

That the node should be deleted, or that ngboost should be revived — #725 records it losing
a 27-month A/B by −3.78 APY pts / −0.14 Sharpe, and nothing here revisits that. That
`min_val_mu_ic: 0` was ever intended to bind; a threshold of 0 with no reader is equally
consistent with "deliberately inert". That these fields are unread at *runtime* — I measured
source references, not execution.
